modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp

```diff
@@ -82,6 +82,7 @@ ParallelCoordinates::ParallelCoordinates()
           TransferFunction{
               {{0.0, vec4{1, 0, 0, 1}}, {0.5, vec4{1, 1, 0, 1}}, {1.0, vec4{0, 1, 0, 1}}}}}
 
+    , multipleAxisSelection_("multipleAxisSelection", "Multiple Axis Selection", false)
```

Comment:
Should it be off by default? This changes the old behavior

Reply:
That's the question! Would it be a more common case to make multiple selections?

Reply:
I don't think so. I guess if one wants to highlight the selected axis in a scatter plot matrix or so. But perhaps it is best to have the more general as default anyway?